lib/errorHelpers.ts

```diff
@@ -0,0 +1,26 @@
+const getConnectError = (error: unknown): string => {
+  const rawErrorMsg =
+    error instanceof Error ? error.message.toLowerCase() : String(error).toLowerCase();
```

Comment:
Is `String(error)` the same as `error.toString()`?

Reply:
`toString()` is not available (at least from TS point of view) since that `error` is type `unknown`. I tested that `String()` works for these errors and I think every type can be safely cast this way

Reply:
okay cool, thanks